ct-app/tools/utils.py

```diff
@@ -70,6 +70,8 @@ def running_module(uppercase: bool = False):
     :returns: the name of the module that is running"""
     argv = sys.argv[0]
 
+    print(sys.argv)
```

Comment:
This could be a log.

Reply:
Or could be deleted completely actually. Done in a new commit ;)